### ISSUE TITLE
Problem with password decryption

### ISSUE THREAD

**Issue by gwaramadze** [jwilk/mbank-cli#90]
Hi, thanks in advance for a great tool. I believe it is great, and can't wait to actually get it to work :D

First, `mbank-cli configure` with GPG encrypted password:
```
$ mbank-cli configure
Country (CZ, or PL, or SK): PL
Login: **********
Password: ************
Encrypt password with GnuPG (y/n)? y
gpg: checking the trustdb
gpg: marginals needed: 3  completes needed: 1  trust model: pgp
gpg: depth: 0  valid:   1  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 1u
gpg: next trustdb check due at 2022-03-10
Session cookie store: ~/.local/share/mbank-cli/52888197.cookies
Created directory for session cookie store: ~/.local/share/mbank-cli
Created configuration file: ~/.config/mbank-cli/config
```
And then I go ahead and ask for anything, eg. list, and book error:
```
$ mbank-cli list
gpg: encrypted with 3072-bit RSA key, ID A01D8457304F91E5, created 2020-03-10
      "Remy Gwaramadze <remy@gwaramadze.com>"
gpg: public key decryption failed: Inappropriate ioctl for device
gpg: decryption failed: No secret key
gpg -d failed at /bin/mbank-cli line 868.
	eval {...} called at /bin/mbank-cli line 873
	main::_decrypt_config(HASH(0x5576cf62e000)) called at /bin/mbank-cli line 902
	main::get_config_var(HASH(0x5576cf62e000), "passwordmanager") called at /bin/mbank-cli line 1760
	main::do_login() called at /bin/mbank-cli line 3320
	main::main() called at /bin/mbank-cli line 3348
```
Could you please help me with this? My bet is that I need to pass the gpg passphrase to the command, but how? Documentation is silent about it or did I miss something? Thanks!

**Comment by jwilk** [comment]
Does 1e2b62cf3a41df7d93e4f4389684c96e06467f22 help?

**Comment by gwaramadze** [comment]
Yes! Thanks.

**Comment by jwilk** [comment]
Yay, thanks for testing.

I'll make a new release later today.

**Comment by jwilk** [comment]
Fixed in [2.2.3](https://github.com/jwilk/mbank-cli/releases/tag/2.2.3).
